<launch>
    <node name="camera_feed" pkg="project_ojakdong" type="camera_feed.py" output="screen">
    </node>
    <node name="follower" pkg="project_ojakdong" type="p_control.py" output="screen">
    </node>
</launch>